substrate/frame/revive/src/impl_fungibles.rs

```diff
@@ -0,0 +1,419 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Implementation of the `fungibles::*` family of traits for `pallet-revive`.
+//!
+//! This is meant to allow ERC20 tokens stored on this pallet to be used with
+//! the fungibles traits.
+//! This is only meant for tests since gas limits are not taken into account,
+//! the feature flags make sure of that.
```

Comment:
```suggestion
//! Implementation of the `fungibles::*` family of traits for `pallet-revive`.
//!
//! This is meant to allow ERC20 tokens stored on this pallet to be used with
//! the fungibles traits.
//! This is only meant for tests since gas limits are not taken into account,
//! the feature flags make sure of that.

#![cfg(any(feature = "std", feature = "runtime-benchmarks", test))]
```